src/main/java/driver/DriverFactory.java

```diff
@@ -1,10 +1,7 @@
 package driver;
 
-import org.openqa.selenium.By;
-import org.openqa.selenium.Keys;
-import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.*;
```

Comment:
please do not use wildcard imports. https://stackoverflow.com/questions/3348816/intellij-never-use-wildcard-imports